OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -27,7 +29,10 @@ public class WithDecorationInfo : UpgradableTraitInfo, ITraitInfo
 		[Desc("Palette to render the sprite in. Reference the world actor's PaletteFrom* traits.")]
 		public readonly string Palette = "chrome";
 
-		[Desc("Pixel offset relative to the top-left point of the actor's bounds.")]
+		[Desc("Point in the actor's bounding box used as reference for offsetting the decoration image.")]
+		public readonly ReferencePoints ReferencePoint = ReferencePoints.TopLeft;
```

Comment:
With enums I find it's always nice to mention the possible values in the `Desc`. Maybe that would be better done with a change to the utility's `--docs` command, though, so feel free to ignore my rambling.